### ISSUE TITLE
Restore sensor state after   reboot

### ISSUE THREAD

**Issue by pterodaktil02** [soloam/ha-pid-controller#33]
Hi
I  have a problem  that   controller forgot it's  state  when  i reboot  my HA and  begin  search  optimal  parametrs from  the begining
It seems a good idia  restore last state  after HA  restart

**Comment by soloam** [comment]
I don't think this is possible, the sensor loses the data on reboot, I don't think it's possible to restore it on reboot! If it's possible I have to see who to implement it.

**Comment by pterodaktil02** [comment]
i  think   most interesting  is I 
 Is there  any way   to read  var  I from  sensor/file  etc?

**Comment by soloam** [comment]
Only I would not be very useful, it would be necessary to also get last input and last error value

**Comment by pterodaktil02** [comment]
Last input data   is  not  critical  -  in stable  PID controller   use only I .
So   after  start
Let's  check  situstion   that  i  have on  any reboot:
I  control    objects   with   very big  time constant (from 5  min  to 3 hous ). 
So  after  hours  of work  controller  found  optimal I  and  it have very small P and D. 
As  i see  -  lost of P  or D    is  not critical   - i will  get them  in  few minutes.  we  can calculate  them  after start.
But we will search for the value of I again within hours.

**Comment by KrzysztofHajdamowicz** [comment]
Idea: implement backing up vital data as JSON to `input_text` every PID calculation. Add service `pid_controller.restore` to read backed up state from said `input_text` . Maybe such feature is enough to make this issue fixed?
